src/client/Microsoft.Identity.Client/IAccount.cs

```diff
@@ -33,5 +33,10 @@ public interface IAccount
         /// </summary>
         /// <remarks>Can be null, for example if this account was migrated to MSAL.NET from ADAL.NET v3's token cache</remarks>
         AccountId HomeAccountId { get; }
+
+        /// <summary>
+        /// Account can have multiple tenants. TenantProfiles is a map of tenant id corresponding tenant profile.
+        /// </summary>
+        IDictionary<string, ITenantProfile> TenantProfiles { get; }
```

Comment:
Add a `<remark>`, that only tenants for which a token was acquired will be available in the TenantProfiles property 
In other words, an account can be in n tenants, but `TenantProfiles` can contain m items (m < n), because tokens were only requested for these m tenants (using `.WithTenant()`)